index.bs

```diff
@@ -4895,6 +5072,21 @@ writable stream:
     .catch(e => console.error("Something went wrong!", e));
 </code></pre>
 
+<div class="note" id="note-web-socket-wrapping-examples">
+  This specific style of wrapping a web socket interprets web socket messages directly as <a>chunks</a>.
+  This can be convenient in some cases, for example when <a>piping</a> to a <a>writable stream</a> or <a>transform
```

Comment:
How about saying "This can be a convenient abstraction, ..." instead.

The phrasing "in some cases" sounds a bit too negative to my ears.